app/src/main/java/org/coepi/android/ui/onboarding/binders/OnboardingCardBinder.kt

```diff
@@ -0,0 +1,13 @@
+package org.coepi.android.ui.onboarding.binders
+
+import android.view.View
+import io.reactivex.subjects.Subject
+import org.coepi.android.ui.onboarding.OnboardingCard
+import org.coepi.android.ui.onboarding.OnboardingClickEvent
+
+interface OnboardingCardBinder {
```

Comment:
Do we really need this abstraction? I personally would just use directly the classes. Then you also can pass the concrete card model class without having to add generics here.